src/components/editor/DraftSpec.tsx

```diff
@@ -26,19 +24,18 @@ function DraftSpecEditor() {
     const [draftSpec, setDraftSpec] = useState<DraftSpecQuery | null>(null);
 
     const handlers = {
-        change: (newVal: any) => {
+        change: (newVal: any, catalogName: string) => {
             if (draftSpec) {
                 const newData = {
```

Comment:
For the reason described in the related comment in this file, I believe prefixing this constant with `new` is a little odd.